.jshintignore

```diff
@@ -1,5 +1,7 @@
 **/vendor
-node_modules
 cms/static/js/i18n/**/*.js
 lms/static/js/i18n/**/*.js
+lms/static/lms/js/build.js
+lms/static/lms/js/spec/main.js
```

Comment:
I was having trouble with those files because they are configurations that don't necessarily obey all the JS conventions. In particular, build.js cannot end with a ``;`` due to the way that r.js processes it. I figured it was simplest to say that these configuration files that aren't used at runtime could just skip optimization.